PageObjects/siteEditorHomePage.py

```diff
@@ -15,3 +15,8 @@
 close_button="xpath=//span[contains(text(),'Close')]"
 all_properties_button="xpath=//span[contains(text(),'All Properties')]"
 group_dropdown_list="xpath=//div[1]/div/div[1]/div[2]/div/div[2]/div/div/div[2]"
+tools_configs_system= "xpath=//div[contains(text(),'SYSTEM')]"
+tools_configs_dashm= "xpath=//div[contains(text(),'DASHM') and @unselectable='on']"
+yes_button = "xpath=//span[contains(text(),'Yes')]"
+sfc_min= "xpath=//div[contains(text(),'SFCMin')]//following::td"
```

Comment:
THis gives 20 element so  select the first specifically

Reply:
changed as per discussion